### ISSUE TITLE
Include snippets/partials in content files

### ISSUE THREAD

**Issue by matthiasprieth** [datenstrom/yellow#303]
First of all nice & simple flat cms! :+1: 

**Is it possible to include snippets/partials into content files?**
So like plugins `[youtube fhs55HEl-Gc]`, I would like to change the footer in edit mode i.e. `[partial footer1]` or `[snippet footer1]`
Or do I need to write my own plugin in this case?

**Comment by schulle4u** [comment]
Currently there is no way to directly modify snippet content as part of the markdown content files, therefore navigation and footer are always global. To have e.g. additional footer content displayed above the global footer you can put it in a `content-pagename.php` snippet file and create a new template called `pagename.html` for it. As an example, open the file `system/themes/snippets/content-default.php`, and modify it to your needs, then save it as `content-pagename.php` in the same folder, where `pagename` is the folder or file name of the page you'd like to apply your modifications to. Now open `system/themes/templates/default.html` and save this as `pagename.html`. Put the following into it: 

```
<?php $yellow->snippet("header") ?>
<?php $yellow->snippet("content-pagename") ?>
<?php $yellow->snippet("footer") ?>
```

However as stated above, footer content in this case is still the same as on other pages, so I think this example only makes sense to e.g. include additional php content without writing a plugin. 

By the way there's a small [plugin written by me](https://github.com/schulle4u/yellow-plugin-global) to make page content globally available in other pages. This is useful to create page teasers and global sidebars. 

HTH
Steffen

**Comment by matthiasprieth** [comment]
Hi Steffen,

Thx for the quick response! :)
I just finished my small plugin when you wrote the message. It's really similar to your `global-plugin`. But yours is even better, cause it's based on content/pages, so i can directly edit them on the frontend e.g. `/yellow/edit/global/sidebar`. Awesome thanks! :+1: 

just one last question: I've set a new template in `config/config.ini`. When I'm logged in on the frontend. I see in the HTML that my template `template-mytemplate` is used. But when I'm logged out it still uses the `template-default` template.
So where can i set it right? There is `Template: xxx` & `TemplateNew: xxx` in `config.ini` but I'm a bit confused with those settings.
